lib/reek/source/source_locator.rb

```diff
@@ -8,64 +8,27 @@ module Source
     # Finds Ruby source files in a filesystem.
     #
     class SourceLocator
+      include Enumerable
+      extend Forwardable
+
+      def_delegator :sources, :each
```

Comment:
No, I don't think that's where each is used. That's the `to_a` for SourcePath, not SourceLocator.